articles/app-service/includes/quickstart-php/quickstart-php-azure-portal-pivot.md

```diff
@@ -0,0 +1,200 @@
+
+## Deploy to Azure
+
+### Sign in to Azure portal
+
+Sign in to the Azure portal at https://portal.azure.com.
+
+
+### Create Azure resources
+
+1. At the top of the portal, type **app services** in the search box. Under **Services**, select **App Services**.
+
+ ![In the Azure portal, search for app services. Then select App Services.](../../media/quickstart-php/azure-portal-search-for-app-services.png)
+
+2. On the **App Services** page, select **Create**.
+
+ ![In the Azure portal, on the App Services page, select Create from the actions bar.](../../media/quickstart-php/azure-portal-create-app-service.png)
+
+3. In the **Basics** tab, under **Project details**, ensure the correct subscription is selected and then select to **Create new** resource group. Type *myResourceGroup* for the name. Then, select **OK**.
+
+ ![On the Create Web App screen, on the Basics tab, in the Project Details section, for Resource group, select the 'Create new' link and add a group named 'myResourceGroup'.](../../media/quickstart-php/azure-portal-create-app-service-create-resource-group.png)
+
+4. Under **Instance details**, set the following settings:
+   - **Name**: Type a globally unique name for your web app. 
+   - **Publish**: Select *Code*.
+   - **Runtime stack**: Select *PHP 8.0*. 
+   - **Operating system**: Select *Linux*.
+   - **Region**: Select the region you want to serve your app from.
+
+ ![On the Create Web App screen, on the Basics tab, in the Instance Details section, use a globally unique name. Select the options for Publish, Runtime stack, Operating system, and region.](../../media/quickstart-php/azure-portal-create-app-service-instance-details-php8.png)
+
+5. Under **App Service Plan**, select **Create new** App Service Plan. Type *myAppServicePlan* for the name. Then select **OK**.
+
+ ![On the Create Web App screen, on the Basics tab, in the App Service Plan section, select the 'Create new' link. Enter myAppServicePlan in the textbox.](../../media/quickstart-php/azure-portal-create-app-service-create-app-service-plan.png)   
+
+6.  To change to the Free tier, next to **Sku and size**, select **Change size**. 
+   
+7.  In the Spec Picker, select **Dev/Test** tab, select **F1**, and select the **Apply** button at the bottom of the page.
+
+ ![In the Spec Picker, select Dev/Test, then select F1. Finally select the Apply.](../../media/quickstart-php/azure-portal-create-app-service-select-free-tier.png)   
+
+    
+8. Select the **Review + create** button at the bottom of the page.
+
+ ![At the bottom of the Create Web App screen, select 'Review + create'.](../../media/quickstart-php/azure-portal-create-app-service-review-create.png)   
+
+
+9. After validation runs, select the **Create** button at the bottom of the page.
+
+10. After deployment is complete, select **Go to resource**.
+
+### Set up continuous deployment
+
+This step will set up continuous deployment using GitHub actions.
+
+1. In the Azure portal, navigate to the app service.
```

Comment:
since this is a continuation you don't need this step. Just continue with step 2: "From the left navigation, select ..."